thrift/thrift0.12/src/test/thrift/TTextProtocolTest.thrift

```diff
@@ -104,7 +104,7 @@ struct TTextProtocolTestMsg {
 
   22: required list<TestUnion> x;
 
-  23: Moji y;  // Does not support string values.
+  23: Moji y;  // Does not support string values for thrift version < 0.19.
```

Comment:
Is there any test case for this from 0.19?

Reply:
The test at https://github.com/line/armeria/pull/5822#discussion_r1689777662 has been modified to better reflect this change